darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -325,7 +323,7 @@ def predict_step(
         batch: TorchInferenceBatch,
         batch_idx: int,
         dataloader_idx: Optional[int] = None,
-    ) -> Sequence[TimeSeries]:
+    ) -> tuple[torch.Tensor, Any, Any]:
```

Comment:
Wouldn't it be possible to have something more informative about the type of `batch_series_schemas` and `batch_pred_starts`?